flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerBase.java

```diff
@@ -311,7 +311,6 @@ public void invoke(IN next) throws Exception {
 		producer.send(record, callback);
 	}
 
-	@Override
```

Comment:
Why did you remove `@Override`?

Reply:
That must have happened by accident, fixing...